testing/test_kerns.py

```diff
@@ -26,6 +26,60 @@ def test_1d(self):
         self.assertTrue(np.allclose(gram_matrix, reference_gram_matrix))
 
 
+class TestArcCosine(unittest.TestCase):
+    def evalKernelError(self, D, variance, weight_variances,
```

Comment:
I copied the name from the test below. Should I change it there, too, although it is not strictly part of the PR?